deploy/cr.yaml

```diff
@@ -68,6 +69,7 @@ spec:
 #        operator: Equal
 #        value: connection-poolers
 #    pgbackrest:
+#      initImage: perconalab/percona-postgresql-operator:main
```

Comment:
It is the wrong place. It should be 
```
spec:
    backups:
        pgbackrest:
            initImage:
```

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1098/commits/1a5669a2b7385cc21f7bd2a1631af6058a721171